apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -44,113 +44,162 @@ export function InstructorAssessmentSettings({
-                </tr>
-                <tr>
-                  <th scope="row">Type</th>
-                  <td>${resLocals.assessment.type}</td>
-                </tr>
-                <tr>
-                  <th scope="row">Set</th>
-                  <td>
-                    ${resLocals.assessment_set.name}
-                    <span class="text-muted">(${resLocals.assessment_set.abbreviation})</span>
-                  </td>
-                </tr>
-                <tr>
-                  <th scope="row">Number</th>
-                  <td>${resLocals.assessment.number}</td>
-                </tr>
-                <tr>
-                  <th scope="row">Module</th>
-                  <td>
-                    ${resLocals.assessment_module
-                      ? html`
-                          ${resLocals.assessment_module.heading}
-                          <span class="text-muted"> (${resLocals.assessment_module.name}) </span>
-                        `
-                      : html` &mdash; `}
-                  </td>
-                </tr>
-                <tr>
-                  <th scope="row">AID</th>
-                  <td>
-                    <span class="pr-2">${resLocals.assessment.tid}</span>
-                    ${resLocals.authz_data.has_course_permission_edit &&
+            <div class="card-body">
+              <form>
+                <div class="form-group">
+                  <label for="title">Title</label>
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="title"
+                    name="title"
+                    value="${resLocals.assessment.title}"
+                    disabled
+                  />
+                  <small class="form-text text-muted"> The title of the assessment. </small>
+                </div>
+                <div class="form-group">
+                  <label for="type">Type</label>
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="type"
+                    name="type"
+                    value="${resLocals.assessment.type}"
+                    disabled
+                  />
+                  <small class="form-text text-muted">
+                    The type of the assessment. This can be either Homework or Exam.
+                  </small>
+                </div>
+                <div class="form-group">
+                  <label for="set">Set</label>
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="set"
+                    name="set"
+                    value="${resLocals.assessment_set.name} (${resLocals.assessment_set
+                      .abbreviation})"
+                    disabled
+                  />
+                  <small class="form-text text-muted"> The set this assessment belongs to. </small>
```

Comment:
Small potential quality-of-life improvement: you could link to the assessment sets page here (e.g. http://localhost:3000/pl/course_instance/4/instructor/course_admin/sets); ditto for the modules description. E.g.:

> The [assessment set](http://localhost:3000/pl/course_instance/4/instructor/course_admin/sets) this assessment belongs to.

Reply:
Added link for assessment sets in [bc4d472](https://github.com/PrairieLearn/PrairieLearn/pull/10653/commits/bc4d472126bb435bb77d4f70e1f2dcd3a45b0912). I did not add it here but do we want to add a `target="_blank"` so this opens in a new tab?

Also, as discussed on Slack, we currently do not have a page to link to for modules but I have opened an issue for this (#10658) and can add a link here after we get that page added.

Reply:
I don't think this needs to open in a new tab. Thanks for opening that issue!